feature-odyssey/src/main/kotlin/com/hedvig/android/odyssey/search/SearchViewModel.kt

```diff
@@ -0,0 +1,44 @@
+package com.hedvig.android.odyssey.search
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.hedvig.android.odyssey.model.ItemType
+import com.hedvig.android.odyssey.model.SearchableClaim
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+
+class SearchViewModel(
+  private val getClaimEntryPoints: GetClaimEntryPoints,
+) : ViewModel() {
+  private val searchableClaims = emptyList<SearchableClaim>()
+  private val _viewState = MutableStateFlow(SearchViewState(results = searchableClaims))
```

Comment:
This isn't used anywhere else aside for the initial status of the _viewState. Do you think it'd be clearer if it was inlined there instead
```suggestion
  private val _viewState = MutableStateFlow(SearchViewState(results = emptyList<SearchableClaim>()))
```
or would you find that confusing? This is a bit of a personal preference, no super strong feelings for it, but I would've inlined it I think.